File: Paneleo.API/Paneleo.Models/Model/Product/Product.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Paneleo.Models.Model.Product
{
    public class Product : Entity
    {
        public string Name { get; set; }
        public string Brand { get; set; }
        public double Quantity { get; set; }
        public string UnitOfMeasure { get; set; }
        public string Vat { get; set; }
        public double NetPrice { get; set; }
        public double GrossPrice { get; set; }
    }
}

File: Paneleo-SPA/src/app/_models/product.ts
export class Product {
  id: number;
  name: string;
  brand: string;
  quantity: number;
  unitOfMeasure: string;
  netPrice: number;
  grossPrice: number;
}

export class ProductForOrder extends Product {
  productQuantity: number;
  totalNetPrice: number;
  totalGrossPrice: number;
}
